main.js

```diff
@@ -0,0 +1,143 @@
+    ],
+    // row3
+    [{ className: 'CapsLock', CapsLock: 'Caps Lock' },
+        { className: 'KeyA', en: ['a', 'A'] },
+        { className: 'KeyS', en: ['s', 'S'] },
+        { className: 'KeyD', en: ['d', 'D'] },
+        { className: 'KeyF', en: ['f', 'F'] },
+        { className: 'KeyG', en: ['g', 'G'] },
+        { className: 'KeyH', en: ['h', 'H'] },
+        { className: 'KeyJ', en: ['j', 'J'] },
+        { className: 'KeyK', en: ['k', 'K'] },
+        { className: 'KeyL', en: ['l', 'L'] },
+        { className: 'Semicolon', en: [';', ':'] },
+        { className: 'Quote', en: ['\'', '"'] },
+        { className: 'Enter', Enter: 'Enter' }
+    ],
+    // row4
+    [{ className: 'ShiftLeft', Shift: 'Shift' },
+        { className: 'KeyZ', en: ['z', 'Z'] },
+        { className: 'KeyX', en: ['x', 'X'] },
+        { className: 'KeyC', en: ['c', 'C'] },
+        { className: 'KeyV', en: ['v', 'V'] },
+        { className: 'KeyB', en: ['b', 'B'] },
+        { className: 'KeyN', en: ['n', 'N'] },
+        { className: 'KeyM', en: ['m', 'M'] },
+        { className: 'Comma', en: [',', '<'] },
+        { className: 'Period', en: ['.', '>'] },
+        { className: 'Slash', en: ['/', '?'] },
+        { className: 'ArrowUp', up: '▲' },
+        { className: 'ShiftRight', Shift: 'Shift' }
+    ],
+    // row5
+    [{ className: 'ControlLeft', Ctrl: 'Ctrl' },
+        { className: 'Meta', Win: 'Win' },
+        { className: 'AltLeft', Alt: 'Alt' },
+        { className: 'Space', Space: 'Space' },
+        { className: 'AltRight', Alt: 'Alt' },
+        { className: 'ControlRight', Ctrl: 'Ctrl' },
+        { className: 'ArrowLeft', left: '◄' },
+        { className: 'ArrowDown', down: '▼' },
+        { className: 'ArrowRight', right: '►' }
+    ],
+];
+
+class KeyboardKey {
+    constructor(options) {
+        const keyButton = document.createElement('div');
+        keyButton.classList.add('keyboard-key');
+        keyButton.classList.add(options.className);
+        if (Object.prototype.hasOwnProperty.call(options, 'en') && options.className !== 'Language') {
+            keyButton.insertAdjacentHTML('afterbegin', options.en[0]);
+
+        } else {
+            keyButton.classList.add('functionKeys');
+            keyButton.insertAdjacentHTML('afterbegin', Object.entries(options)[1][1]);
+        }
+        return keyButton;
+    }
+}
+
+function createKeyboardRow(numberOfRow) {
+    const keyboardRow = document.createElement('div');
+    keyboardRow.classList.add('keyboard-row');
+    keyboardDesc[numberOfRow].forEach((option) => {
+        const newKey = new KeyboardKey(option);
+        keyboardRow.insertAdjacentElement('beforeend', newKey);
+    });
+    return keyboardRow;
+}
+
+function createKeyboard() {
+    const keyboard = document.createElement('div');
+    keyboard.classList.add('keyboard');
+    for (let i = 0; i < keyboardDesc.length; i += 1) {
+        const newRow = createKeyboardRow(i);
```

Comment:
Inside method `createKeyboardRow` you again take data from `keyboardDesc` by index, in my opinion will be better if you just provide `keyboardDesc[i]` as argument for `createKeyboardRow` method.